server/config/seed.js

```diff
@@ -31,3 +31,42 @@ const insertSeed = async () => {
 };
 
 insertSeed();
+import pool from './index';
+
+(async () => {
```

Comment:
Since this file already has a seed not written as IIFE's, I would also suggest that it's either these new seeds follow the old style or the old one be rewritten to follow the async IIFE style so that we will be consistent in function styles used in the file.

Reply:
@obayomi96 actually worked on this aspect.